Yoast/wordpress-seo
# Use cornerstone assessors for cornerstone content

## Summary
This pull needs https://github.com/Yoast/YoastSEO.js/pull/1141 (make sure that pull has been merge before merging this one)

This PR can be summarized in the following changelog entry:

## Test instructions

This PR can be tested by following these steps:

* Make sure to use `https://github.com/Yoast/YoastSEO.js#1139-add-cornerstone-assessor` in the package.json for yoastseo.
* Create or edit a post
* Check the cornerstone content checkbox and see the analysis result will be changed.

Fixes #7103
